allennlp/models/coreference_resolution/coref.py

```diff
@@ -141,6 +146,13 @@ def forward(self,  # type: ignore
             A tensor of shape ``(batch_size, num_spans_to_keep)`` representing, for each top span, the
             index (with respect to antecedent_indices) of the most likely antecedent. -1 means there
             was no predicted link.
+        antecedent_scores : ``torch.FloatTensor``, optional
+            A tensor of shape ``(batch_size, num_spans_to_keep, 1 + max_antecedents)`` giving
+            the antecedent scores for each mention.  Each i-th batch element is associated with a
+            matrix whose the j-th row contains the antecedent scores for the j-th mention of that
+            batch (corresponding to top_spans).  The column k-th column contains the score for
+            the (k-1)-th mention being the antecedent of the j-th mention. The first column
+            (index 0) contains the score for the j-th mention having no antecedent.
```

Comment:
To make sure we are talking about the matrix that results from indexing into the tensor:
```suggestion
            (index k = 0) contains the score for the j-th mention having no antecedent.
```